core/clingutils/src/dequeLinkdef.h

```diff
@@ -3,10 +3,11 @@
 
 #pragma extra_include "deque";
 
+#pragma create TClass deque<short>;
 #pragma create TClass deque<int>;
 #pragma create TClass deque<long>;
 #pragma create TClass deque<float>;
 #pragma create TClass deque<double>;
+#pragma create TClass deque<long double>;
```

Comment:
We do not provide the `long double` dictionary for any other collection.  If we do here, we ought to also add it to for the other collection.  If we do add it to any collection we also need to double check that it fails 'nicely'/'clearly' if the user try to store them (`long double` is not support in the I/O)